### PR TITLE
Reverted etcd template

### PR DESCRIPTION
**Pull request by tusharraut1994**
ETCD snapshotter does not work with CSI and it always needs storage class which support shared volume creation.

Reverted ETCD template to not use snapshotter.

### CODE REVIEW

File: charts/px-backup/templates/px-backup/pxcentral-etcd.yaml [hunk 1]
```diff
@@ -394,19 +276,19 @@ spec:
...
+              value: "http://$(MY_POD_NAME).pxc-backup-etcd-headless:2380"
             - name: ETCD_LISTEN_PEER_URLS
               value: "http://0.0.0.0:2380"
             - name: ETCD_INITIAL_CLUSTER_TOKEN
               value: "pxc-backup-etcd-cluster"
             - name: ETCD_INITIAL_CLUSTER_STATE
               value: "new"
             - name: ETCD_INITIAL_CLUSTER
-              value: "pxc-backup-etcd-0=http://pxc-backup-etcd-0.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain}}:2380,pxc-backup-etcd-1=http://pxc-backup-etcd-1.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain}}:2380,pxc-backup-etcd-2=http://pxc-backup-etcd-2.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain}}:2380"
+              value: pxc-backup-etcd-0=http://pxc-backup-etcd-0.pxc-backup-etcd-headless:2380,pxc-backup-etcd-1=http://pxc-backup-etcd-1.pxc-backup-etcd-headless:2380,pxc-backup-etcd-2=http://pxc-backup-etcd-2.pxc-backup-etcd-headless:2380,
```
Comment (diptiranjanpx) [inline charts/px-backup/templates/px-backup/pxcentral-etcd.yaml:291]: ',' is there at the end.

File: charts/px-backup/templates/px-backup/pxcentral-etcd.yaml [hunk 2]
```diff
@@ -168,7 +121,9 @@ data:
 
     # Constants
     AUTH_OPTIONS=""
-    export ETCDCTL_ENDPOINTS="pxc-backup-etcd-0.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:2380,pxc-backup-etcd-1.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:2380,pxc-backup-etcd-2.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:2380"
+    ETCDCTL_ENDPOINTS="http://pxc-backup-etcd-0.pxc-backup-etcd-headless:2379,http://pxc-backup-etcd-1.pxc-backup-etcd-headless:2379,http://pxc-backup-etcd-2.pxc-backup-etcd-headless:2379,"
```
Comment (diptiranjanpx) [inline charts/px-backup/templates/px-backup/pxcentral-etcd.yaml:124]: should the port be 2380? also there is a "," at the end.

File: charts/px-backup/templates/px-backup/pxcentral-etcd.yaml [hunk 3]
```diff
@@ -23,45 +23,28 @@ data:
...
 
     # Constants
     HOSTNAME="$(hostname -s)"
     AUTH_OPTIONS=""
-    export ETCDCTL_ENDPOINTS="pxc-backup-etcd-0.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:2380,pxc-backup-etcd-1.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:2380,pxc-backup-etcd-2.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:2380"
-    export ROOT_PASSWORD="${ETCD_ROOT_PASSWORD:-}"
-    if [[ -n "${ETCD_ROOT_PASSWORD:-}" ]]; then
-      unset ETCD_ROOT_PASSWORD
-    fi
+    ETCDCTL_ENDPOINTS="http://pxc-backup-etcd-0.pxc-backup-etcd-headless:2379,http://pxc-backup-etcd-1.pxc-backup-etcd-headless:2379,http://pxc-backup-etcd-2.pxc-backup-etcd-headless:2379,"
```
Comment (diptiranjanpx) [inline charts/px-backup/templates/px-backup/pxcentral-etcd.yaml:37]: "," is there at the end.

File: charts/px-backup/templates/px-backup/pxcentral-etcd.yaml (hunk 1 above)
Comment (tusharraut1994) [reply charts/px-backup/templates/px-backup/pxcentral-etcd.yaml]: removed

File: charts/px-backup/templates/px-backup/pxcentral-etcd.yaml (hunk 2 above)
Comment (tusharraut1994) [reply charts/px-backup/templates/px-backup/pxcentral-etcd.yaml]: removed

File: charts/px-backup/templates/px-backup/pxcentral-etcd.yaml (hunk 3 above)
Comment (tusharraut1994) [reply charts/px-backup/templates/px-backup/pxcentral-etcd.yaml]: removed

**Review by diptiranjanpx — APPROVED** [review]

**Review by sharma-tapas — APPROVED** [review]

**Pull request merged**